containers/python/xgboost_container_entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
```

Comment:
Modify the permissions of this file via `sudo chmod 755 xgboost_container_entry.sh` and commit the result.